python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -124,13 +129,131 @@ def setDefaults(self):
         self.badMaskPlanes = ["NO_DATA", "BAD", "CR",]
 
 
+## \addtogroup LSST_task_documentation
+## \{
+## \page AssembleCoaddTask
+## \ref AssembleCoaddTask_ "AssembleCoaddTask"
+## \copybrief AssembleCoaddTask
+## \}
+
 class AssembleCoaddTask(CoaddBaseTask):
-    """Assemble a coadd from a set of coaddTempExp
+    """!
+\anchor AssembleCoaddTask
+
+\brief Assemble a coadded image from a set of coadded temporary exposures.
+
+\section pipe_tasks_assembleCoadd_Contents Contents
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Purpose
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Initialize
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Run
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Config
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Debug
+  - \ref pipe_tasks_assembleCoadd_AssembleCoaddTask_Example
+
+\section pipe_tasks_assembleCoadd_AssembleCoaddTask_Purpose	Description
+
+\copybrief AssembleCoaddTask_
+
+We want to assemble a coadded image from a set of coadded temporary exposures (coaddTempExps). We implicitly 
+assume that the coaddTempExps were all observed in the same filter band. Each input coaddTempExp 
```

Comment:
I think the code is generic enough that we do not assume that the coaddTempExps are from the same filter.  I would remove this comment.